### PR TITLE
Don't cancel builds on main branch

### PR DESCRIPTION
**Pull request by alexeagle**
Upstream https://github.com/aspect-build/rules_jasmine/pull/129

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -12,8 +12,8 @@ on:
 
 concurrency:
   # Cancel previous actions from the same PR: https://stackoverflow.com/a/72408109
```
Comment (gregmagolan) [inline .github/workflows/ci.yaml:14]: I iterated on the comment a bit. Seen here: https://github.com/aspect-build/rules_js/pull/1502/files

**Review by gregmagolan — CHANGES_REQUESTED** [review]
See comments

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (alexeagle) [reply .github/workflows/ci.yaml:14]: need to update rules_jasmine too?

**Comment by alexeagle** [comment]
updated PTAL

**Review by gregmagolan — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (gregmagolan) [reply .github/workflows/ci.yaml]: rules_jasmine has the latest incarnation as well now https://github.com/aspect-build/rules_jasmine/blob/ef94648234b75a7726da71fc2dca5334096a9321/.github/workflows/ci.yaml#L14

**Pull request merged**